src/mount/writedata.cc

```diff
@@ -675,7 +676,8 @@ void* write_worker(void*) {
 
 /* API | glock: INITIALIZED,UNLOCKED */
 void write_data_init(uint32_t cachesize, uint32_t retries, uint32_t workers,
-		uint32_t writewindowsize, uint32_t chunkserverTimeout_ms, uint32_t cachePerInodePercentage) {
+		uint32_t writewindowsize, uint32_t chunkserverTimeout_ms, 
```

Comment:
Please, remove the trailing space after `uint32_t chunkserverTimeout_ms, `

Reply:
Done